### ISSUE TITLE
Wrong Block In the Broken pieces

### ISSUE THREAD

**Issue by JustBlackHole** [haubna/PhysicsMod#44]
So like when you break a block, it will have the pieces go everywhere, but I noticed when you break a Nether Brick Fence, it breaks into... leaves...
All the pieces are just leaf block pieces

**Comment by JustBlackHole** [comment]
Okay so I kept on breaking blocks after I posted this and it seems like it's not just nether brick fence... I'm in the nether and I'm just tearing down a fortress with the collapse thing, and it seems as if there's a phantom tree or something in the blocks. I kept breaking and started seeing wood pieces too. I feel like this might have to do with the fact that I'm also using the Immersive Portals mod, and maybe it thinks I'm breaking the blocks that are around my nether portal in the overworld, or something.
